### DISCUSSION TITLE
Inventory Movement Report

### DISCUSSION THREAD

**Discussion by tomd-tc** [inventree/InvenTree#8414] [General]
Howdy!

I've just started using Inventree and I'm a little lost.

My CPA would like to see the $ value of material depletion (as a result of it being sold) for October. I tracked all incoming material, the builds made (from the incoming material) and sales. Is there a way to export these reports for a given month - without exporting and manipulating the data in Excel?

**Comment by SchrodingersGat** [comment]
Hey @tomd-tc  we have the [inventree-order-history plugin](https://github.com/inventree/inventree-order-history) which might be what you are looking for here? It can display (and export) order history across all parts, or a particular part, or a given company, etc...

**Comment by tomd-tc** [comment]
Hey @SchrodingersGat . I tried installing it per instructions but at reboot, I saw the plugin didn't initiate. In the Plugin Error section I see the following:

**Stage** discovery
**Name** order_history
**Message** cannot import name 'UserInterfaceMixin' from 'plugin.mixins' (/path/to/inventree/src/backend/InvenTree/plugin/mixins/__init__.py)

Would you have any suggestions what could be wrong? 

Current version details:
InvenTree Version: 0.16.0 dev
API Version: 232
Python Version: 3.11.2
Django Version: 4.2.14

P.S. We really like your product!








InvenTree Version	[0.16.0 dev](https://github.com/inventree/InvenTree/releases)API Version	[232](http://wms.pulcro.lan/api-doc/)
Python Version	3.11.2
Django Version	[4.2.14](https://www.djangoproject.com/)

**Reply by SchrodingersGat** [reply]
> Would you have any suggestions what could be wrong?

Ah, that one is on me. I should have mentioned that the plugin uses the new UI plugin system, which is only available on the upcoming `0.17.x` release. For "stable" (currently 0.16.7) it won't work

However if you're currently running `0.16.0 dev` (rather than the `stable` branch) you can update to the latest dev version and the plugin should work
